src/autogen.ml

```diff
@@ -0,0 +1,107 @@
+let mk_input dir input =
+  Filename.concat dir input
+
+let mk_mapper file =
+  let mapper_name = "mapper_" ^ file ^ ".native" in
+  let path_to_bin = Filename.dirname Sys.argv.(0) in
+  Filename.concat path_to_bin mapper_name
+
+let mk_output dir file =
+  Filename.concat dir (file ^ ".ml")
+
+(* template.ml is what the student is going to write into. Therefore, we want
+ * to modify a bit the output of the pretty-printer to add a white line between
+ * each function of the exercise. We also remove trailing whitespaces. *)
+let change_template_fill output template_fill =
+  if Sys.command ("sed -Ei 's/[ \\t]+$//' " ^ output) <> 0 then
```

Comment:
This comment is needed because you did not introduce a function named "remove_trailing_whitespaces" and a function named "insert_line_between_each_functions".